paradox-nakadi-consumer-boot/src/main/java/de/zalando/paradox/nakadi/consumer/boot/ReplayHandler.java

```
@@ -31,21 +33,25 @@
 
     private static final PartitionCoordinator THROWING_COORDINATOR = new EmptyPartitionCoordinator() {
         @Override
-        public void error(final Throwable t, final EventTypePartition eventTypePartition) {
+        public void error(final Throwable t, final EventTypePartition eventTypePartition, @Nullable final String cursor,
+                final String rawEvent) {
+
+            // todo should we add the handler?
```

Error handler should be only added if the exception is not handled elsewhere.